tools/circle_plus_gen/README.md

```diff
@@ -0,0 +1,40 @@
+# Circle+ generator
+
+If a circle file has training hyperparameters, we usually call it a 'circle+' file.<br/>
+This tool generates a circle+ file by injecting training hyperparameters into a circle file.<br/>
+It also helps handle circle+ file, such as checking whether the circle file contains training hyperparameters. <br/> 
+
+## Requirements
+
+1. (optional) Set python virtaul environment.
+
+    This tool tested on python3.8.
+
+    ```
+    python3 -m venv venv
+    source /venv/bin/activate
+    ```
+
+2. Install required pakcages. 
+
+    Currently, only `flatbuffers==24.3.25` is needed.
+    ```bash
+    pip install -r requirements.txt
+    ```
+
+## Inject training parameters using json file
```

Comment:
The word `parameters` may cause confusion with model parameters. How about modify `parameter` to `hyperparameters`?

Reply:
ditto
https://github.com/Samsung/ONE/pull/12977/files#diff-61170ff4427e0fe0c44fb9f9d8b479f00400606d936c9fd7535062e0cccd78edR33-R34